haskell-indexer-frontend-kythe/src/Language/Haskell/Indexer/Frontend/Kythe.hs

```diff
@@ -206,6 +207,15 @@ makeRelationFacts (Relation src relKind target) =
            -- So by squinting a bit, we'll hopefully get better functionality
            -- out of Kythe-related tools that are not aware of Haskell.
 
+-- | Makes fact about module imports.
+makeImportFacts :: ModuleTick -> Conversion [Raw.Entry]
+makeImportFacts ModuleTick{..} = do
+    vn <- asks baseVName
+    makeAnchor mtSpan RefImportsE (pkgvn vn) Nothing Nothing
```

Comment:
please use the name portion of `makePackageFacts` (or factor it) to get the package vname.